### PR TITLE
Fix namespace(@mapbox) issue in tilelive.auto

### PR DESCRIPTION
**Pull request by jingsam**

### CODE REVIEW

**Comment by jingsam** [comment]
The tests seem unstable, success on node 0.10 and node 4, failed on node 0.12 and node 5

File: lib/tilelive.js [hunk 1]
```diff
@@ -109,7 +109,7 @@ tilelive.auto = function(uri) {
     uri.protocol = uri.protocol || keyword + ':';
 
     if (!tilelive.protocols[uri.protocol]) {
-        [keyword, 'tilelive-' + keyword].forEach(function(name) {
+        [keyword, 'tilelive-' + keyword, '@mapbox/tilelive-' + keyword].forEach(function(name) {
```
Comment (mapsam) [inline lib/tilelive.js]: This is great @jingsam! The only thing I worry about is making the assumption that a tilelive module is under the `@mapbox` namespace. It definitely solves a majority of the modules, but there are others that could still hit the issue. Any ideas here?

Comment (jingsam) [inline lib/tilelive.js]: What's the specific issue?

**Comment by tcql** [comment]
lets figure out what's up the with the tests and I'll get this merged

**Comment by mapsam** [comment]
The tests appear to be giving different results when run multiple times. Not great, but shouldn't block this fix. Feel free to pull the trigger @tcql!

**Comment by tcql** [comment]
yea seems like the test indeterminism isn't related to this change 👍  thanks for the fix @jingsam

**Pull request merged**